app/root.css

```diff
@@ -192,145 +207,212 @@ h2 {
 }
 
 .padding-bottom-4 {
-  padding-bottom: var(--spacing-4);
+  padding-bottom: var(--spacing-4) !important;
```

Comment:
I don't like overusing important ... but if no one has time to remove the wrong padding from selectors with higher priority that overwrite these classes, it's not a blocker 😭